### PR TITLE
Add a sentence to contact section

### PR DESCRIPTION
**Pull request by Hiresh23**
## Purpose
To add lines in the contact sections
#84

## Goals
It will provide a way for site viewers to get in touch

## Approach
Just a section with necessary text and apply related css

### Screenshots
Screenshot 2022-10-15 114828

  
### Preview Link

https://pr-105-new-yoshitha-website.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
For the issue number in the purpose don't put it in <> tags

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Put the preview link as this https://pr-105-NEW-YOSHITHA-WEBSITE.surge.sh

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Change the topic of this pr as the topic of the issue

**Comment by Hiresh23** [comment]
Done

**Comment by Hiresh23** [comment]
Check now

File: dark/assets/css/styles.css [hunk 1]
```diff
@@ -923,3 +931,47 @@ html {
     bottom: -26px;
     left: calc(50% - 20px);
 }
+
+#contact-section .context1{
+    width: 550px;
+    margin: 90px 100px; 
+    position: relative;
+    left: 380px;
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:939]: Remove the left and align the text center

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 2]
```diff
@@ -479,7 +479,7 @@ html {
     margin-top: 20px;
     border: 2px solid var(--primary-color);
     border-radius: 10px;
-    padding: 30px 40px;
+    padding: 18px 40px;
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:482]: Undo these unwanted changes

File: dark/assets/css/styles.css [hunk 3]
```diff
@@ -602,12 +602,15 @@ html {
 .social-media-icons {
     position: relative;
     top: 300px;
-    margin: 50px;
+    margin: 80px;
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:605]: Undo these unwanted changes

File: dark/assets/css/styles.css [hunk 4]
```diff
@@ -602,12 +602,15 @@ html {
 .social-media-icons {
     position: relative;
     top: 300px;
-    margin: 50px;
+    margin: 80px;
     margin-bottom: 400px;
     display: flex;
-    margin-left: 460px;
+    /* margin-left: 460px; */
+    justify-content: center;
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:609]: Undo these unwanted changes

File: dark/assets/css/styles.css [hunk 5]
```diff
@@ -602,12 +602,15 @@ html {
...
     top: 300px;
-    margin: 50px;
+    margin: 80px;
     margin-bottom: 400px;
     display: flex;
-    margin-left: 460px;
+    /* margin-left: 460px; */
+    justify-content: center;
+}
+.ix{
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:611]: Undo these unwanted changes

File: dark/assets/css/styles.css [hunk 6]
```diff
@@ -811,12 +814,13 @@ html {
...
 
-.second-section {
+/* instead of single section made contact and about two different sections */
+/* .second-section {
     box-shadow: 0 0 50px rgba(0, 0, 0, 1);
     background-color: rgba(0, 0, 0, 0.9);
     backdrop-filter: blur(10px);
-    height: 1000px;
-}
+    height: 000px;
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:822]: Undo these unwanted changes

File: dark/assets/css/styles.css [hunk 7]
```diff
@@ -877,7 +881,12 @@ html {
     font-size: 20px;
 }
 
-
+#about-section{
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:884]: Undo these unwanted changes

File: dark/assets/css/styles.css [hunk 8]
```diff
@@ -888,7 +897,6 @@ html {
     font-size: 90px;
     filter: blur(4px);
     font-family: impact;
-    z-index: -1;
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:891]: Undo these unwanted changes

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 9]
```diff
@@ -923,3 +931,47 @@ html {
...
+    background-color: rgba(0, 0, 0, 0.9);
+    backdrop-filter: blur(10px);
+    text-align: center;
+}
+
+
+
+
+/* form related css  */
+input ,label{
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:958]: There isn't a form so remove these unwanted codes

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 10]
```diff
@@ -157,55 +157,55 @@ <h1><span class="first-text">I'm a </span><span class="auto-type"></span></h1>
         <div id="tooltip">
           <span id="tooltiptext">Whatsapp</span>
           <a href="https://api.whatsapp.com/send/?phone=0714345661" target="_blank "
-            ><img src="https://img.icons8.com/color/48/000000/whatsapp--v1.png" alt="Whatsapp Icon"
+            ><img class ="ix" src="https://img.icons8.com/color/48/000000/whatsapp--v1.png" alt="Whatsapp Icon"
```
Comment (YoshithaRathnayake) [inline dark/index.html:160]: Remove these unwanted changes

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Remove the languages image

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Resolve the conflicts

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
@Hiresh23,

- First, delete this branch and the pr.
- Create a new branch and checkout it.
- After that git remote add upstream https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE.git
- git fetch upstream
- git reset --hard upstream/main
- git pull upstream main
- After that do the changes.
- Add the changes and commit them
- git push origin BRANCH_NAME

**Comment by Hiresh23** [comment]
Please remove spam

**Comment by Hiresh23** [comment]
I could not complete because my exam are going

**Pull request closed without merging**